test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-provisioner.yaml

```diff
@@ -40,6 +40,7 @@ spec:
       serviceAccountName: csi-provisioner
       containers:
         - name: csi-provisioner
+          #image: docker.io/xyang105/csi-provisioner:apiv1
```

Comment:
For the provisioner arg error, I think we need to remove https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/utils/deployment.go#L92

Reply:
I see.  That makes sense now.